Dockerfile

```diff
@@ -2,13 +2,20 @@ FROM quay.io/centos/centos:stream8
 
 RUN dnf -y module install python39 && dnf -y install python39 python39-pip
 RUN mkdir /app
-ADD https://raw.githubusercontent.com/arcalot/arcaflow-plugin-template-python/main/LICENSE /app
-ADD example_plugin.py /app
-ADD test_example_plugin.py /app
-ADD requirements.txt /app
+RUN chmod 777 /app
+ADD https://raw.githubusercontent.com/arcalot/arcaflow-plugin-template-python/main/LICENSE /app/
+ADD README.md /app/
+ADD poetry.lock /app/
+ADD pyproject.toml /app/
+ADD example_plugin.py /app/
+ADD test_example_plugin.py /app/
+RUN chmod +x /app/example_plugin.py /app/test_example_plugin.py
 WORKDIR /app
 
-RUN pip3 install -r requirements.txt
+RUN pip3 install poetry
+RUN poetry config virtualenvs.create false
+RUN poetry install --without dev
```

Comment:
We should probably use `python3.9 -m pip` and  `python3.9 -m poetry` to prevent it from using python 3.6, which doesn't work on newer versions of poetry.

Reply:
will do.